src/AppRoutes.tsx

```diff
@@ -0,0 +1,26 @@
+import {
+  HashRouter as Router,
+  Route,
+  Routes,
+  Navigate,
+} from 'react-router-dom';
+import { NotFoundPage } from './NotFoundPage';
+import { HomePage } from './HomePage';
+import { PeoplePage } from './PeoplePage.1';
+import { App } from './App';
+
+export const Root = () => (
```

Comment:
Please name components and the corresponding file the same way. Other way it becomes confusing